src/main/java/Duke.java

```diff
@@ -1,10 +1,101 @@
+import java.util.Scanner;
+
 public class Duke {
-    public static void main(String[] args) {
+    public static final int MAX_TASKS = 100;
+
+    public static String getFirstWord(String word){
+        String firstWord = word;
+        if (firstWord.contains(" ")) {
+            firstWord = firstWord.substring(0, firstWord.indexOf(" "));
+        }
+        return firstWord;
+    }
+
+    public static boolean checkOneWord(String word){
```

Comment:
Since this is a boolean method, you should use a boolean sounding name to be more clear.